std/getopt.d

```diff
@@ -401,6 +401,27 @@ getopt(args,
 An unrecognized option such as "--baz" will be found untouched in
 `args` after `getopt` returns.
 
+Treat an option like some additional help option:
+If an application needs to add some own option, which is treated like the
+--help (-h) option and therefore circumvents the checks for required arguments,
+it can pass the `std.getopt.config.helpLike` directive to `getopt`:
```

Comment:
I'm not a fan of the name 'helpLike', it took me a while to realize what it actually does. My first impression was that it would print a (help) text, but it's not doing any printing, it's just a regular callback option. Only after your response to Paul did it click that all it does is circumventing required arguments, so why not call it something like `circumventRequired`?

The documentation can still explain that it's useful for things like `--help` or `--version`, but the way it's currently described in the changelog and documentation really threw me off.

Reply:
Thanks for the feedback. I was pretty unsure how to name this option. The option circumvents the required checks AND also sets the help wanted flag. As said above, it allows for additional options, which operates like the --help argument.
That is why I named it  `helpLike`. The name  `circumventRequired` does not imply, that the help flag is set as well - it only circumvents the any previous `required`option. So not exactly the intended meaning.
Any other proposals for a good/better name?

Reply:
> AND also sets the help wanted flag

Is this really needed? If you have more than one help-like argument (such as both `--version` and `--help`), you'll need to use booleans to distinguish them anyway.